mri/operators/linear/test_wavelet_adjoint.py

```diff
@@ -107,7 +107,8 @@ def test_Wavelet_UD_2D_Multichannel(self):
         multichannel case"""
         for i in range(self.max_iter):
             print("Process Wavelet Undecimated test '{0}'...", i)
-            wavelet_op = WaveletUD2(nb_scale=4, multichannel=True, n_cpu=2)
+            wavelet_op = WaveletUD2(nb_scale=4, n_coils=self.num_channels,
```

Comment:
Try to use my technique to avoid future unnecessary changes

Reply:
Yeah, will update